apps/music_proxy.cpp

```diff
@@ -39,13 +39,17 @@
 
 namespace po = boost::program_options;
 
+namespace
+{
+const double defaultMusicTimestep = 0.0001;
+const brion::URI pluginURI( MONSTEER_BRION_SPIKES_PLUGIN_SCHEME + "://" );
```

Comment:
The other apps in this project don't do it, and I don't remember doing it anywhere else myself. As far as I know, the _ were for private class member, but I don't have a strong opinion on that. We should just try to be consistent.